Cargo.toml

```diff
@@ -12,7 +12,6 @@ name = "lib"
 harness = false
 
 [features]
-debug-layout = []
```

Comment:
FWIW I removed it because it stopped being useful to me and it didn't compile anymore.